gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
But that's bad news bears -- site-nav is included in most of the other
pages on the site, that could be runtime cached. Don't we need to list them
too?
On Tue, Jun 7, 2016 at 7:27 AM Arthur Evans <notifications@github.com>
wrote:

> In gulpfile.js
> <https://github.com/Polymer/docs/pull/1641#discussion_r66080878>:
>
> > @@ -82,6 +83,58 @@ function createReloadServer() {
> >    });
> >  }
> >
> > +function writeServiceWorkerFile() {
> > +  var path = require('path');
> > +  var rootDir = 'dist';
> > +
> > +  var config = {
> > +    cacheId: 'polymerproject',
> > +    staticFileGlobs: [
> > +      rootDir + '/1.0/index.html',
>
> Oh, I had totally missed directoryIndex. That explains a bit that I was
> confused about.
>
> dist is the build directory, so I think:
>
> dynamicUrlToDependencies: {
>   '1.0/': [ 'dist/1.0/index.html', 'dist/templates/head-meta.html', 'dist/templates/site-nav.html' ]
> }
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Polymer/docs/pull/1641/files/ca126abbb1148b4c112496b094bed83b53c3af25#r66080878>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABTkUuj9BqHgtQqWPoo7lL65fNLRDOKUks5qJX_GgaJpZM4Is_dF>
> .
>